### PR TITLE
test: Add revive test with puppeteer

### PR DESCRIPTION
**Pull request by sylc**
towards https://github.com/lucacasonato/fresh/issues/164

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,6 +35,14 @@ jobs:
       - name: Cache dependencies
         run: deno cache --no-check src/cli/deps.ts src/runtime/deps.ts src/server/deps.ts
 
+      - name: Setup puppeteer linux / mac
+        if: startsWith(matrix.os, 'ubuntu') || startsWith(matrix.os, 'mac')
+        run: PUPPETEER_PRODUCT=chrome deno run -A --unstable https://deno.land/x/puppeteer@9.0.2/install.ts
+
+      - name: Run linter
```
Comment (lucacasonato) [inline .github/workflows/ci.yml]: Name is incorrect.

File: tests/deps.ts [hunk 2]
```diff
@@ -7,3 +7,5 @@ export {
   TextLineStream,
 } from "https://deno.land/std@0.129.0/streams/delimiter.ts";
 export { delay } from "https://deno.land/std@0.129.0/async/delay.ts";
+import puppeteer from "https://deno.land/x/puppeteer@9.0.2/mod.ts";
+export default puppeteer;
```
Comment (lucacasonato) [inline tests/deps.ts]: why default export?

**Review by lucacasonato — COMMENTED** [review]
Looks great, just a minor comment.

**Comment by sylc** [comment]
Fixed.

**Review by lucacasonato — APPROVED** [review]
Thanks!

**Pull request merged**